src/modules/navigator/rtl_params.c

```diff
@@ -105,12 +105,14 @@ PARAM_DEFINE_FLOAT(RTL_LAND_DELAY, -1.0f);
 PARAM_DEFINE_FLOAT(RTL_MIN_DIST, 5.0f);
 
 /**
```

Comment:
1. The docs and QGC call this mode "Return". If possible we should do the same. The point being that not every return is a land - e.g. for FW you return and circle.
2. Value 1 seems odd in the light of value 2 - ie what if value 1 is set but no mission landing point is available? Also what if value 2 is set but person would prefer to land at the starting point. Also, what if the mission has no land command for both these cases - ie is just end of mission or a loiter?

Maybe for this block (assuming value 2 is as I thought. 
```
/**
 * Return type
 *
 * Flight path and destination in the event of Return (RTL) being invoked.
 *
 * @value 0 Return to home position via direct path.
 * @value 1 Return to planned mission landing point.
 * @value 2 Return to planned mission landing point (if available) or reverse mission path and land at take off point.
 * @group Return
 */
```